lib/reflect/controls/target.rb

```diff
@@ -0,0 +1,95 @@
+module Reflect
+  module Controls
+    module Target
+      def self.example(constant_name: nil, subject: nil)
+        constant_name ||= Namespace::Random.get
+        subject ||= Subject.example
+
+        target =
+          Module.new do
+            extend Methods
+          end
+
+        subject_constant = Reflect.constant(subject)
+        subject_constant.const_set(constant_name, target)
+
+        target
+      end
+
+      module Methods
+        def self.extended(cls)
+          cls.extend(SomeMethod)
+          cls.extend(SomeAccessor)
+          cls.extend(SomeObjectAccessor)
+        end
+
+        module SomeMethod
+          def some_method(some_parameter, some_other_parameter)
+          end
+        end
+
+        module SomeAccessor
+          def self.extended(cls)
+            some_inner_module = Module.new
+            cls.const_set(:SomeInnerConstant, some_inner_module)
+          end
+
+          def some_accessor
+            some_inner_module = const_get(:SomeInnerConstant)
+            some_inner_module
```

Comment:
The difference between `SomeInnerConstant` and `some_inner_module` needs an explanation (or to be harmonized if there isn't a need for a difference)